commons/passive/google/google_sleep_classify_event.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.passive.google",
+  "type": "record",
+  "name": "GoogleSleepClassifyEvent",
+  "doc": "Sleep classification event including the sleep confidence, device motion and ambient light level.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "timeReceived", "type": "double", "doc": "Device receiver timestamp in UTC (s)." },
+    { "name": "sleepConfidence", "type": "int", "doc": "Sleep confidence value between 0 and 100." },
```

Comment:
you can remove the `sleep` prefix. I'm not sure what the confidence means. Confidence of what? In general, for probability values please use a float with range 0 - 1.

Reply:
Actually, here confidence means sleep confidence as mentioned [here](https://developers.google.com/android/reference/com/google/android/gms/location/SleepClassifyEvent#:~:text=public%20int-,getConfidence,-()).

Reply:
Ah OK. Can you please add the full docstring in the link to here (except the range)? Because this provides a reason of why it is called sleepConfidence.